src/util/sss_log.c

```diff
@@ -59,13 +62,22 @@ static int sss_to_syslog(int priority)
 
 static void sss_log_internal(int priority, int facility, const char *format,
                              va_list ap);
+#endif
+
 
 void sss_log(int priority, const char *format, ...)
 {
     va_list ap;
 
     va_start(ap, format);
+#if !defined(WITH_STDERR_SYSLOG)
     sss_log_internal(priority, LOG_DAEMON, format, ap);
+#else
```

Comment:
Just an idea and I'm not sure it is worth the effort: instead of using a conditional build here, you could simply redefine the `sss_log_internal()` function to do all these `*printf()` calls.

Reply:
Ok, done.